Userland/Libraries/LibJS/Runtime/Temporal/Calendar.cpp

```diff
@@ -867,7 +867,6 @@ ThrowCompletionOr<ISOMonthDay> iso_month_day_from_fields(VM& vm, Object const& f
     return ISOMonthDay { .month = result->month, .day = result->day, .reference_iso_year = reference_iso_year };
 }
 
-// 12.2.37 ISOYear ( temporalObject ), https://tc39.es/proposal-temporal/#sec-temporal-isoyear
 i32 iso_year(Object& temporal_object)
 {
     // 1. Assert: temporalObject has an [[ISOYear]] internal slot.
```

Comment:
From the commit I see that these functions no longer exist in the spec.
However they still have spec comments and are used in the code.
I think we should at least remove the spec comments to indicate this method is not specced. Although definitely wait for confirmation from @linusg or others since I don't know if there is precedent for this.

Secondly perhaps it would actually possible to rewrite the code to not use these functions/not make them public by removing them from (I assume) Calendar.h.

Reply:
Very simple - if these are still used in LibJS, implement the spec change that removes the use first. If they are unused, obviously removed the full functions :)